subsys/net/lib/nrf_cloud/src/nrf_cloud_transport.c

```diff
@@ -219,7 +219,12 @@ static int endp_send(const struct nct_dc_data *dc_data, struct mqtt_utf8 *endp,
 	if (qos != MQTT_QOS_0_AT_MOST_ONCE) {
 		publish.message_id = get_message_id(dc_data->message_id);
 	}
-
+	LOG_DBG("mqtt_publish: id = %d len = %d, topic: %.*s", publish.message_id,
```

Comment:
maybe we should add a separate nrf_cloud MQTT and MQTT traffic logging option (in another PR) for better logging control.

Reply:
Added IRIS-9483